packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -486,8 +486,9 @@ rocksdb::Status ArbCore::reorgToMessageCountOrBefore(
         std::cerr << "Loaded valid checkpoint, gas: " << output.arb_gas_used
                   << ", messages: " << output.fully_processed_inbox.count
                   << ", steps: " << output.total_steps
-                  << ", logs: " << output.log_count << ", sends"
-                  << output.send_count << "\n";
+                  << ", logs: " << output.log_count
+                  << ", sends: " << output.send_count
+                  << ", gas: " << output.arb_gas_used << "\n";
```

Comment:
This seems to double print gas used (it's printed at the start too).

Reply:
fixed